Components/Button/button.css

```diff
@@ -0,0 +1,158 @@
+    --button-icon-size: 1.4rem;
+    --floating-button-icon-size: 1.5rem;
+    /* focus color is seen when you use tab key to access buttons or links */
+    --primary-button-focus-outline-color: black;
+    --secondary-button-focus-outline-color: #bae6fd;
+    --warn-button-focus-outline-color: black;
+    --primary-link-focus-outline-color: black;
+    --secondary-link-focus-outline-color: #bae6fd;
+}
+
+.button {
+    cursor: pointer;
+    padding: 0.2em 1em;
+    font-size: 1.5rem;
+}
+
+.button.button-primary {
+    background-color: var(--primary-button-bg-color);
+    border: var(--primary-button-border);
+    border-radius: var(--primary-button-border-radius);
+    color: var(--primary-button-color);
+}
+
+.button.button-primary:focus {
+    outline: var(--primary-button-focus-outline-color) solid 2px;
+}
+
+.button.button-secondary {
+    background-color: var(--secondary-button-bg-color);
+    border: var(--secondary-button-border);
+    border-radius: var(--secondary-button-border-radius);
+    color: var(--secondary-button-color);
+}
+
+.button.button-secondary:focus {
+    outline: var(--secondary-button-focus-outline-color) solid 2px;
+}
+
+.button.button-warn {
+    background-color: var(--warn-button-bg-color);
+    border: var(--warn-button-border);
+    border-radius: var(--warn-button-border-radius);
+    color: var(--warn-button-color);
+}
+
+.button.button-warn:focus {
+    outline: var(--warn-button-focus-outline-color) solid 2px;
+}
+
+.link {
+    text-decoration: none;
+    padding: 0.2em 1em;
+    font-size: 1.5rem;
+}
+
+.link.link-primary {
+    background-color: var(--primary-link-bg-color);
+    border: var(--primary-link-border);
+    border-radius: var(--primary-link-border-radius);
+    color: var(--primary-link-color);
+}
+
+.link.link-primary:focus {
+    outline: var(--primary-link-focus-outline-color) solid 2px;
+}
+
+.link.link-secondary {
+    background-color: var(--secondary-link-bg-color);
+    border: var(--secondary-link-border);
+    border-radius: var(--secondary-link-border-radius);
+    color: var(--secondary-link-color);
+}
+
+.link.link-secondary:focus {
+    outline: var(--secondary-link-focus-outline-color) solid 2px;
+}
+
+.button.button-icon-container {
+    padding-left: 0.8em;
+
+    /* flex - properties */
+    display: flex;
+    align-items: center;
+    gap: 0.5em;
+}
+
+.button-svg-icons {
+    font-size: var(--button-icon-size) !important;
+}
+
+.button-primary .button-svg-icons {
+    color: var(--primary-button-icon-color);
+}
+
+.button-secondary .button-svg-icons {
+    color: var(--secondary-button-icon-color);
+}
+
+.button-warn .button-svg-icons {
+    color: var(--warn-button-icon-color);
+}
+
+.button.button-floating {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    border-width: 1px; /* thicker border distorts circle shape */
+
+    /* flex - properties */
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
you can make utility class for this ... you used it many times

Reply:
Yes i have made the file for utilities, would implement this everywhere later after other work. Noted the suggestions in a notion page.